README.md

```diff
@@ -47,27 +46,27 @@ view of a file with very long lines. More example pairs can be found in the
 
 ## Version support
 
-The latest version of `golines` requires golang 1.18 or newer due to generics-related dependencies.
+Newer releases of `golines` require golang 1.21 or newer due to generics-related dependencies.
```

Comment:
Very minor nit: The need for 1.21 isn't specifically for generics-related dependencies like 1.18 or higher is.

Reply:
I didn't feel like rewriting a chunk of the README last night. Let's see if that changes today.

Reply:
README updated